docs: simplify component descriptions (#294)
arcus-azure/arcus: website/resources/components/web-api.md

---
title: Arcus Web API

> description: Simplified API development with shared access key authentication, HTTP correlation, request tracking, strict JSON formatting, and other boilerplate code that can be added with zero effort. 

Reviewer: ```suggestion
description: Simplify your API development with shared access key authentication, HTTP correlation, request tracking, strict JSON formatting, and other boilerplate code.
```
